examples/case_studies/reliability_and_calibrated_prediction.myst.md

```diff
@@ -444,23 +447,6 @@ def bayes_boot(df, lb, ub, seed=100):
 ```
 
 ```{code-cell} ipython3
-def bootstrap(df, lb, ub, seed=100):
```

Comment:
This is the same function as above, and this one wasn't being used, so I deleted it.

Reply:
It was a slightly different function. One implemented the naive bootstrap and the other implemented the bayesian bootstrap with a flat dirichelet prior. But yeah, basically the same, so i'm fine getting rid of it.